test/cloud-metadata/callback-coordination.js

```diff
@@ -1,8 +1,16 @@
 'use strict'
 const tape = require('tape')
+// mock logger
+const logger = {
+  error: function () {
+  },
+  debug: function () {
+  }
+}
+
```

Comment:
If you want a no-op logger, you can get one via:

```
const logging = require('../logging')
logging.createLogger('off')
```

See https://github.com/elastic/apm-agent-nodejs/blob/master/lib/logging.js#L62-L69

Reply:
✅ done